### ISSUE TITLE
Axe not flagging any error/warning in the attached scenarios when using aira-labelledby /aria-describedby

### ISSUE THREAD

**Issue by PavaniHFP** [dequelabs/axe-core#2993]
Axe is currently not flagging an error/warning when an IDREF listed as a value for an aria-describedby and/or aria-labelledby attribute is not found in the DOM. 

1. When there are multiple IDREFs listed axe does NOT throw a warning/error if at least one IDREF is found in the DOM for ( aria-describedby /aria-labelledbyattributes)

2. axe does not throw a warning/error when aria-describedby' attribute do not refer any ID in DOM

3. axe does not throw a warning/error if the values for the ID's were not found in the DOM for aria-describedby

Attached excel includes my results and my sample code pen pages 

https://codepen.io/pavanigutta/pen/zYZRzoe

[axe analysis.xlsx](https://github.com/dequelabs/axe-core/files/6650334/axe.analysis.xlsx)

https://codepen.io/pavanigutta/pen/yLMbjLp

**Comment by straker** [comment]
Thanks for the issue. The short answer is that axe-core is not a linter and only checks for accessibility violations. 

Since only one `aria-labelledby` is needed to create an accessible name, we only need one to determine if it's a violation or not. Moreover, not all screen readers support multiple values in `aria-labelledby`, as shown in this codepen https://codepen.io/straker/pen/PvqONy, so we can only look for at least one.

Since `aria-describedby` does not contribute to an accessible name, we don't check it for existing DOM nodes. This is especially true since there are valid reasons to have an `aria-describedby` that doesn't point to an element currently that may exist later (such as an error message for an invalid input).
